packages/components/src/input-control/types.ts

```diff
@@ -67,6 +67,23 @@ export type InputChangeCallback<
 	P = {}
 > = ( nextValue: string | undefined, extra: { event: E } & P ) => void;
 
+export type WithTooltipProps = {
+	/**
+	 * React children.
+	 */
+	children: ReactNode;
+	/**
+	 * Whether or not to wrap children in a Tooltip component.
+	 *
+	 * @default false
+	 */
+	showTooltip?: boolean;
+	/**
+	 * Text to display within the Tooltip component.
```

Comment:
Probably forgot to update this after copy-pasting

Reply:
Similar to my [reply](https://github.com/WordPress/gutenberg/pull/42726#discussion_r939839376) above, this description does reflect the prop's purpose for `WithTooltip`.